radicle/src/git.rs

```diff
@@ -79,17 +76,36 @@ pub mod refs {
     pub mod storage {
         use super::*;
 
+        /// Where the project's identity document is stored.
+        ///
+        /// `refs/rad/id`
+        ///
+        pub static IDENTITY_BRANCH: Lazy<Qualified> = Lazy::new(|| {
```

Comment:
nit: a branch is technically something that is under `refs/heads` or `refs/remotes`, I believe.

Reply:
Hmmm, yeah I've seen it called branch in a couple places in link I think. I do think that it's more descriptive than `IDENTITY_REF`, and it is "branch-like", but perhaps let's see if it's confusing people and change it then.

Reply:
Ya, good points! :) Let's leave it as-is then

Reply:
Eg. in identity-resolution.adoc:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/40774/199261471-72f038ef-5a23-402b-9ff9-ac726450ba3e.png">